libraries/tools/kotlin-formver/src/common/kotlin/org/jetbrains/kotlin/formver/gradle/FormVerExtension.kt

```diff
@@ -7,8 +7,13 @@ package org.jetbrains.kotlin.formver.gradle
 
 open class FormVerExtension {
     internal var myLogLevel: String? = null
+    internal var myUnsupportedFeatureBehaviour: String? = null
 
     open fun setLogLevel(logLevel: String) {
         myLogLevel = logLevel
     }
+
+    open fun setUnsupportedFeatureBehaviour(behaviour: String) {
+        myUnsupportedFeatureBehaviour = behaviour
+    }
```

Comment:
Why don't we use Kotlin's setters? Check [here](https://kotlinlang.org/docs/properties.html#backing-fields)

Reply:
We're using this from Gradle, so not using properties is the idiomatic way; see for example how Lombok does it: https://github.com/jesyspa/kotlin/blob/formal-verification/libraries/tools/kotlin-lombok/src/common/kotlin/org/jetbrains/kotlin/lombok/gradle/LombokExtension.kt